tests/e2e/tests/simple/simple1_test.go

```diff
@@ -110,6 +128,33 @@ func TestSvc2Svc(t *testing.T) {
 	if len(podList) != 2 {
 		t.Fatalf("Unexpected to get %d pods when expecting 2. got %v", len(podList), podList)
 	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
I would move that whole section into a function so it can be re-used but that can be a follow up PR